package.json

```diff
@@ -73,7 +73,7 @@
     "npm-run-all": "^4.1.5",
     "opn": "^5.3.0",
     "precise-commits": "1.0.2",
-    "prettier": "1.10.2",
+    "prettier": "1.15.3",
```

Comment:
`prettier` was updated so that it could handle `html` files during `format-files`